src/wwwroot/controllers/PlanCtrl.js

```diff
@@ -35,7 +36,8 @@
     vm.planInfoLoader = true;
     vm.planStatusInfoLoader = true;
     vm.ipsPlanCount = 0;
-
+    vm.errorsPlanUpgrade = isUpgradeablePlan(); //DR-1514
```

Comment:
Im not sure why we need to have the Jira ticket number in the code.

Reply:
Will be deleted. I am currently using it to quickly recognize my implementations. Thanks @mvillaseco